server/src/main/java/com/broadcom/lsp/cobol/core/visitor/CobolVisitor.java

```diff
@@ -63,10 +63,14 @@
   private static final String INVALID_DEF_MSG = "Invalid definition for: ";
   private static final String MISSPELLED_WORD = "A misspelled word, maybe you want to put ";
   private static final String PROGRAM_ID_ISSUE_MSG = "There is an issue with PROGRAM-ID paragraph";
+  private static final String PARAGRAPH_NOT_DEFINED_MSG = "The following paragraph is not defined: ";
```

Comment:
we have i18n service now, should be externalized

Reply:
Great, can you point me to an example of how to do that?

Reply:
MessageService#getMessage(key)

Reply:
@Nurkambay probably, if you pull changes from dev branch, these would be unused we can get rid of them. Apart from the newly introduced. For which you would have to introduce your key in messages_en.properties file and use.

Reply:
Thanks! Done.